Code/Editor/MainWindow.cpp

```diff
@@ -845,78 +852,190 @@ void MainWindow::InitActions()
         .SetShortcut(tr("Z"))
         .SetToolTip(tr("Center on Selection (Z)"))
         .Connect(&QAction::triggered, this, &MainWindow::OnGotoSelected);
+
+    const auto goToViewBookmarkFn = [](int index)
+    {
```

Comment:
We can do this but as it's not used anywhere else right now and captures no local state this would be easy to refactor in future. I am certain we would move this for the 'shared' camera work, but as that isn't hooked up right now this doesn't seem critical to me to change